raiden_contracts/utils/logs.py

```diff
@@ -167,3 +171,7 @@ def uninstall(self):
         assert self.filter is not None
         self.web3.eth.uninstallFilter(self.filter.filter_id)
         self.filter = None
+
+
+# A concrete event added in a transaction.
+LogRecorded = namedtuple('LogRecorded', 'message callback count')
```

Comment:
Maybe move this to the top, this is hard to find otherwise

Reply:
Yes, doing in https://github.com/raiden-network/raiden-contracts/pull/759